tip-distributor/src/bin/reclaim-rent.rs

```diff
@@ -0,0 +1,52 @@
+//! Reclaims rent from TDAs and Claim Status accounts.
+
+use {
+    clap::Parser,
+    log::*,
+    solana_client::nonblocking::rpc_client::RpcClient,
+    solana_sdk::{pubkey::Pubkey, signature::read_keypair_file},
+    solana_tip_distributor::reclaim_rent_workflow::reclaim_rent,
+    std::{path::PathBuf, str::FromStr},
+    tokio::runtime::Runtime,
+};
+
+#[derive(Parser, Debug)]
+#[clap(author, version, about, long_about = None)]
+struct Args {
+    /// RPC to send transactions through.
+    #[clap(long, env)]
+    rpc_url: String,
+
+    /// Tip distribution program ID.
```

Comment:
can you add comment here that the RPC should have indices enabled for this program? otherwise will kill it